code/modules/surgery/organs/liver.dm

```diff
@@ -45,11 +47,39 @@
 	if(damage > maxHealth)//cap liver damage
 		damage = maxHealth
 
+	if(swelling >= 10)
+		pharmacokinesis()
+
 /obj/item/organ/liver/prepare_eat()
 	var/obj/S = ..()
 	S.reagents.add_reagent("iron", 5)
 	return S
 
+//Just in case
+/obj/item/organ/liver/Remove(mob/living/carbon/M, special = 0)
+	M.remove_movespeed_modifier("pharma")

```

Comment:
movespeed modifiers ids need to all be defined.

Reply:
Think I've done it right, is it okay to define it in the liver.dm?